fw_upgrade/README.md

```diff
@@ -159,17 +164,17 @@ vol_size=1
 
 Make the image:
 
-```
-ubinize -p 128KiB -m 2048 -o ubiimage ubiini
+```bash
+$ ubinize -p 128KiB -m 2048 -o ubiimage ubiini
 ```
 
 2. Package the ubi image:
 
 ```
-mkimage -f fw.its ubiimage_packaged
+$ mkimage -f fw.its ubiimage_packaged
 ```
 
-Here is `fw.its`:
+Here's the `fw.its` file that you can use:
```

Comment:
Are there any changes required to the `fw.its` file? I guess the `timestamp`?

Reply:
not required.